frontend/src/scenes/insights/__mocks__/funnelLeftToRight.json

```diff
@@ -33,7 +33,8 @@
             "type": "AND",
             "values": []
         },
-        "funnel_viz_type": "steps"
+        "funnel_viz_type": "steps",
+        "sampling_factor": 0.1
```

Comment:
I assume it's ok to keep these on some graphs? wouldn't want a whole new story just changing this filter but having some views with and some without is nice?